### DISCUSSION TITLE
wis2box 1.0b2 released

### DISCUSSION THREAD

**Discussion by tomkralidis** [wmo-im/wis2box#424] [Announcements]
This is the second beta release of wis2box. This release includes a number of enhancements and bugfixes.

## Updates

- force error handling in bufr2geojson (#417)
- add optional FTP forwarder service (#297)
- add improved monitoring features (#329)
- run station publishing on startup and every 10 minutes (#413) 
- fix date handling for BUFR pipeline (#412)
- allow csv2bufr templates to be local or external (#405)
- add standard data pipeline reporting (#329)
- update to ecCodes 2.28.0 (#381)
- add SSL deployment option (#140)
- update WIS2 notification message generation (#391)
- align data mappings workflow to stations (#388)
- set broker default message queue to 1000 (#377)
- add public user to broker for WIS2 compliance (#386)
- add WIS2 topic hierrachy validation (#228)

## Migrating existing installations

Given the changes in various configurations, it is recommended to install 1.0b2 from scratch:
- delete wis2box Docker volumes
- update discovery metadata YAML configuration examples
- update `station_list.csv` from examples provided
- repeat runtime configuration

## Documentation
Documentation for this release is available at https://docs.wis2box.wis.wmo.int/en/1.0b2

## Feedback
All contributions are always welcome. Feedback is welcome via GitHub [issues](https://github.com/wmo-im/wis2box/issues) or [discussions](https://github.com/wmo-im/wis2box/discussions).

The wis2box team

https://docs.wis2box.wis.wmo.int

**Comment by lsc346531272** [comment]
Hello,

I'm trying to install, configure and run wis2box 1.0b2 version.
